machine/wnc_x86_64/wnc.patch/installer/x86_64/install-arch

```diff
@@ -0,0 +1,499 @@
+# x86_64 specific ONIE installer functions
```

Comment:
No function is lacking now. It's just used for copy my modify to origin source file(replace it). For example, Assume my ONIE HDD is sda, in general, sda1 for grub-boot, sda2 for onie-boot. and sda3 if for NOS. But I want to create a 2K block sda3 for TLV EEPROM, and sda4 for NOS. So I had to modify origin partition rule. Looks like below:

+# add eeprom partition
+    onie_eeprom_part=$(( $grub_boot_part + 2 )) # sda3
+    onie_eeprom_start=$(( $onie_boot_start + $onie_boot_size )) 
+    # 2KB
+    onie_eeprom_size=$(( 4 )) # 4 = 2K
+# end

That because my developing board do not implement additional EEPROM for store TLV info. So I want to implement TLV EEPROM it at sda. And at last, I will modify i2c code to access /dev/sda3, not access addtional EEPROM via i2c interface.

And my question is:
I spilt ONIE partition from 3 to 4(add TLV EEPROM partition), is it permitted?
Or it's an vendor side issue?

In certification preparation, the code should put in GitHub(pull request) and image should able to download. 
There is a question, if my console speed is 57600, and 
1. if I don't have permit to modify speed to 57600 and pull a request, can I satisfy the certification preparation?
2. if I have permit  speed to 57600 and pull a request, maybe it will effect others.
So, some of file changed for my own purpose, should implement at local side I think.
But my code pull request success on GitHub, will it built at Lab and load to DUT to do testing? or just update DUT image from which we put on server?
If the code should be build and test, I think test will fail if my own feature can not pull a request.
ex, speed=57600, and four MTD partitions(add sda3 for implement TLV EEPROM), and modify i2c code to access TLV EEPROM.